### ISSUE TITLE
"flag provided but not defined: -initial-advertise-peer-urls" on EC2 instance

### ISSUE THREAD

**Issue by imtheghoul** [coreos/etcd#4286]
Hello! Trying to deploy etcd 0.4.9 cluster on AWS EC2 nodes with CoreOS-stable-681.1.0-hvm using this guide: https://coreos.com/etcd/docs/latest/clustering.html#etcd-discovery
My command on first node (192.168.7.9) is:
etcd -name infra0 -initial-advertise-peer-urls http://192.168.7.9:2380 \
  -listen-peer-urls http://192.168.7.9:2380 \
  -listen-client-urls http://192.168.7.9:2379,http://127.0.0.1:2379 \
  -advertise-client-urls http://192.168.7.9:2379 \
  -discovery https://192.168.7.9/v2/keys/discovery/6c007a14875d53d9bf0ef5a6fc0257c817f0fb83
I've got a error: "flag provided but not defined: -initial-advertise-peer-urls".
Did I missed something?

**Comment by xiang90** [comment]
@imtheghoul That flag is introduced in etcd2. etcd 0.4 does not have that flag. I suggest you to upgrade to a later version of coreos that ships etcd2. Thanks!

**Comment by imtheghoul** [comment]
Thank you, that's it! Now I'm using etcd 2.2.4
But now I have another problem. Can I ask here or should I create another issue? Or maybe I should to ask some other way?
The case is: I can't understand how to bootstrap cluster in according to guide https://github.com/coreos/etcd/blob/master/Documentation/clustering.md#etcd-discovery.
When I do "curl -X PUT https://192.168.7.9/v2/keys/discovery/6c007a14875d53d9bf0ef5a6fc0257c817f0fb83/_config/size -d value=2" I have got: "curl: (7) Failed to connect to 192.168.7.9 port 443: Connection refused"
Same for http.
I tried to run etcd before curl with command from first message and also tried just run "etcd &" - curl doesn't works.
Also I don't understand what is this ID 6c007a14875d53d9bf0ef5a6fc0257c817f0fb83. Can I use any ID instead this, or maybe I must to generate it somehow?
